docs/howto/use_podman.md

```diff
@@ -0,0 +1,46 @@
+# How to Develop Using Podman Instead of Docker
+
+Podman is a container engine that is compatible with Docker but does not require a daemon and can run containers rootless. If you want to use Podman for development instead of Docker, follow these steps:
+
+## 1. Install Podman
+
+Refer to the [official Podman installation guide](https://podman.io/getting-started/installation) for your operating system.
+
+## 2. Configure Docker Compatibility
+
+Podman provides a Docker-compatible CLI. You can setup Podman so it runs whenever the docker commands are run, [follow the migration guide](https://podman-desktop.io/docs/migrating-from-docker/managing-docker-compatibility)
+
+## 3. Run Docker Commands with Podman
+
+Most Docker commands work with Podman. For example:
+
+```sh
+podman build -t my-image .
+podman run -it my-image
+podman ps
+```
+
+## 4. Use Podman Compose
+
+If your project uses `docker-compose`, install [podman-compose](https://github.com/containers/podman-compose):
+
+```sh
+pip install podman-compose
+```
+
+Then use:
+
+```sh
+podman-compose up
+```
```

Comment:
I think points 3 and 4 are not needed here, we are not documenting podman itself, but how to use it with elastic-package.